docs/CONTRIBUTING.md

```diff
@@ -0,0 +1,72 @@
+# Contributing
+
+## Contributor License Agreement
+
+In order to contribute, you must accept the [contributor license agreement](https://cla-assistant.io/dequelabs/axe-core-npm) (CLA). Acceptance of this agreement will be checked automatically and pull requests without a CLA cannot be merged.
+
+## Contribution Guidelines
+
+### Code Quality
+
+Although we do not have official coding guidelines, we can and will request you to make changes if we think that your code is sloppy. You can take clues from the existing code base to see what we consider to be reasonable code quality. Please be prepared to make changes that we ask of you even if you might not agree with the request(s).
+
+Pull requests that change the tabs of a file (spacing or changes from spaces to tabs and vice versa) will not be accepted. Please respect the coding style of the files you are changing and adhere to that.
+
+That having been said, we prefer:
+
+1. Tabs over spaces
+2. Single quotes for string literals
+3. Function definitions like `function functionName(arguments) {`
+4. Variable function definitions like `Class.prototype.functionName = function (arguments) {`
+5. Use of 'use strict'
+6. Variables declared at the top of functions
+
+### Documentation and Comments
+
+Functions should contain a preceding comment block with [jsdoc](http://usejsdoc.org/) style documentation of the function. For example:
+
+```
+/**
+ * Runs the Audit; which in turn should call `run` on each rule.
+ * @async
+ * @param  {Context}   context The scope definition/context for analysis (include/exclude)
+ * @param  {Object}    options Options object to pass into rules and/or disable rules or checks
+ * @param  {Function} fn       Callback function to fire when audit is complete
+ */
+```
+
+Classes should contain a jsdoc comment block for each attribute. For example:
+
+```
+/**
+ * Constructor for the result of checks
+ * @param {Object} check CheckResult specification
+ */
+function CheckResult(check) {
+	'use strict';
```

Comment:
I am _strongly_ against opting in to strict mode in function scopes. Either the entire module/file should be in strict mode or none of it.

Again, using ESM makes this moot.